airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -3040,6 +3049,7 @@ abstract class BasicFunctionalityIntegrationTest(
 
     @Test
     open fun testClear() {
+        assumeTrue(verifyDataWriting)
```

Comment:
no reason not to have this here, it wasn't _breaking_ anything, but is technically more correct